index.html

```diff
@@ -16,42 +16,246 @@
+                  <li>javascript</li>
+                </ul>
+              </div>
+              <div class="btn"><button>See Project</button></div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <!-- About my self section -->
+      <section id="about-myself">
+        <div class="container">
+          <div class="heading">
+            <h1>
+              About<br />
+              Myself
+            </h1>
+          </div>
+          <div class="paragraph-one">
+            <p class="paragraph">
+              Hello I’m a software developer! I can help you build a product ,
+              feature or website Look through some of my work and experience! If
+              you like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+            <p class="lets-connect"><a href="#">LET’S CONNECT</a></p>
+          </div>
+          <ul class="social-media-links">
+            <li>
+              <a href="#"><img src="images/twitter.png" alt="twitter" /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="images/Linkedinicon.png" alt="linkedin"
+              /></a>
+            </li>
+            <li>
+              <a href="#"><img src="images/Vector.png" alt="microverse" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="images/gitgub.png" alt="github" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="images/Vector3.png" alt="vector" /></a>
+            </li>
+          </ul>
+          <div class="btn"><button>Get my resume</button></div>
         </div>
-        <div class="paragraph-one">
-          <p>
-            I’m a software developer! I can help you build a product , feature
-            or website Look through some of my work and experience! If you like
-            what you see and have a project you need coded, don’t hestiate to
-            contact me.
-          </p>
-          <P class="lets-connect"><a href="#" >LET’S CONNECT</a></P>
-        </div> 
-        <ul class="social-media-links">
-          <li>
-            <a href="#"><img src="images/twitter.png" alt="twitter"/></a>
-          </li>
-          <li>
-            <a href="#"
-              ><img src="images/Linkedin icon.png" alt="linkedin"
-            /></a>
-          </li>
-          <li>
-            <a href="#"><img src="images/Vector .png" alt="microverse" /></a>
-          </li>
-          <li>
-            <a href="#"
-              ><img src="images/gitgub.png" alt="github" 
-            /></a>
-          </li>
-          <li>
-            <a href="#"><img src="images/Vector (3).png" alt="vector" /></a>
-          </li>
-        </ul>
+        <div class="list-container container">
+          <ul class="list">
+            <li class="dropdown">
+              <div class="union">
+                <span class="langauge-border">Langauge</span
```

Comment:
In the Figma design, there is a drop-down icon but it is not present in yours, kindly let us include it so that we can meet up with the design requirements. 👇

**Your Design**
<img width="382" alt="image" src="https://user-images.githubusercontent.com/73854999/205440861-69aafb87-24c5-4cdd-8b9d-1b3d448879a5.png">

**Figma Design**
<img width="382" alt="image" src="https://user-images.githubusercontent.com/73854999/205440888-484247c3-ff94-405b-9f18-6cb78d244a16.png">

Reply:
- [ ] Kindly address this issue suggested by the previous code reviewer.